server/src/main/java/com/distkv/server/storeserver/runtime/expire/ExpireCycle.java

```diff
@@ -0,0 +1,133 @@
+package com.distkv.server.storeserver.runtime.expire;
+
+import com.distkv.common.exception.DistkvException;
+import com.distkv.rpc.protobuf.generated.DistkvProtocol.DistkvRequest;
+import com.distkv.rpc.protobuf.generated.DistkvProtocol.RequestType;
+import com.distkv.rpc.protobuf.generated.ExpireProtocol.ExpireRequest;
+import com.distkv.server.storeserver.StoreConfig;
+import com.google.protobuf.InvalidProtocolBufferException;
+import java.util.PriorityQueue;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.ReentrantLock;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ExpireCycle {
+
+  private static Logger LOGGER = LoggerFactory.getLogger(ExpireCycle.class);
+
+  private static ScheduledExecutorService swapExpiredPool
+      = new ScheduledThreadPoolExecutor(1);
+
+  private ReentrantLock lock = new ReentrantLock();
```

Comment:
What does this do?

Reply:
Now , i think it's may useless now. I'll remove it.